notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
<p>Change the last sentence to:</p><p>"In this tutorial, the model creation stage of the pipeline costs ...."</p><p> </p><p>How come this is the only pipeline stage that provides a cost estimate?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>You're right, added a link to the pricing page when the user trains a Vertex Forecasting model.</p>